RTCLauncher/RTCLauncher.csproj

```diff
@@ -27,159 +19,42 @@
   </PropertyGroup>
   <PropertyGroup>
     <ApplicationIcon>Resources\rtcpastel.ico</ApplicationIcon>
   </PropertyGroup>
   <PropertyGroup>
     <SignManifests>false</SignManifests>
   </PropertyGroup>
-  <PropertyGroup>
-    <SignAssembly>false</SignAssembly>
-  </PropertyGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
     <Reference Include="System.IO.Compression" />
-    <Reference Include="System.IO.Compression.FileSystem" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Deployment" />
-    <Reference Include="System.Drawing" />
-    <Reference Include="System.Windows.Forms" />
-    <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="DownloadForm.cs">
+    <Compile Update="LaunchPanelV1.cs">
       <SubType>Form</SubType>
     </Compile>
-    <Compile Include="DownloadForm.Designer.cs">
-      <DependentUpon>DownloadForm.cs</DependentUpon>
-    </Compile>
-    <Compile Include="LaunchPanelV3.cs">
+    <Compile Update="LaunchPanelV2.cs">
       <SubType>Form</SubType>
     </Compile>
-    <Compile Include="LaunchPanelV3.Designer.cs">
-      <DependentUpon>LaunchPanelV3.cs</DependentUpon>
-    </Compile>
-    <Compile Include="MainForm.cs">
-      <SubType>Form</SubType>
-    </Compile>
-    <Compile Include="MainForm.Designer.cs">
-      <DependentUpon>MainForm.cs</DependentUpon>
-    </Compile>
-    <Compile Include="LaunchPanelV2.cs">
+    <Compile Update="LaunchPanelV3.cs">
       <SubType>Form</SubType>
     </Compile>
-    <Compile Include="LaunchPanelV2.Designer.cs">
-      <DependentUpon>LaunchPanelV2.cs</DependentUpon>
-    </Compile>
-    <Compile Include="Objects.cs" />
-    <Compile Include="LaunchPanelV1.cs">
+    <Compile Update="MainForm.cs">
       <SubType>Form</SubType>
     </Compile>
-    <Compile Include="LaunchPanelV1.Designer.cs">
-      <DependentUpon>LaunchPanelV1.cs</DependentUpon>
-    </Compile>
-    <Compile Include="Program.cs" />
-    <Compile Include="Components\BuildContextMenu.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Properties\Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-    <Compile Include="RTC_Extensions.cs" />
-    <Compile Include="SidebarVersionsPanel.cs">
+    <Compile Update="SidebarInfoPanel.cs">
       <SubType>Form</SubType>
     </Compile>
-    <Compile Include="SidebarVersionsPanel.Designer.cs">
-      <DependentUpon>SidebarVersionsPanel.cs</DependentUpon>
-    </Compile>
-    <Compile Include="VersionDownloadPanel.cs">
+    <Compile Update="SidebarVersionsPanel.cs">
```

Comment:
The fix for #16 is that we need to explicitly call out files that should be opened by the Designer.

I'm not sure when this happens automatically and when you need to do it manually. This seems to be inconsistent between RTCV and RTC-Launcher. However, being explicit seems to always work.

Reply:
From my understanding VS is supposed to detect it and do it automatically but maybe something was being weird with the upgrade. I bet the reason it was displaying as designable but then crashing on design for me was because of some cache in .VS.